packages/polyfills/src/baseline.ts

```diff
@@ -0,0 +1,6 @@
+import '@babel/polyfill';
+import {auto as unhandledRejectionPolyfill} from 'browser-unhandled-rejection';
```

Comment:
I think it is important because we want people to think about their scheduling and will likely build packages around using it. That said, can definitely wait.